Gems/Atom/Tools/MaterialEditor/Code/Source/Document/MaterialDocument.cpp

```diff
@@ -694,10 +688,8 @@ namespace MaterialEditor
         }
         else if (AzFramework::StringFunc::Path::IsExtension(m_absolutePath.c_str(), MaterialTypeSourceData::Extension))
         {
-            materialTypeSourceFilePath = m_absolutePath;
-
             // Load the material type source data, which will be used for enumerating properties and building material source data
-            auto materialTypeOutcome = MaterialUtils::LoadMaterialTypeSourceData(materialTypeSourceFilePath);
+            auto materialTypeOutcome = MaterialUtils::LoadMaterialTypeSourceData(m_absolutePath);
```

Comment:
It would be helpful to have a comment that explains why, in this case, m_absolutePath refers to a .materialtype file even though we are inside the MaterialDocument class which represents a .material file. The reader would usually assume that this->m_absolutePath refers to a .material

Reply:
Or could m_absolutePath be renamed in some way to make this more clear? Or another variable added so we have m_materialAbsolutePath and m_materialTypeAbsoluatePath?

Reply:
I will add a comment